vision/src/autogluon/vision/detector/detector.py

```diff
@@ -164,6 +168,17 @@ def fit(self,
                 Mini batch size
             lr : float
                 Trainer learning rate for optimization process.
+            early_stop_patience : int, default=-1
+                Number of epochs with no improvement after which train is early stopped. Use -1 to disable.
+            early_stop_min_delta : float, default=1e-4
+                The small delta value to ignore when evaluating the metric. A large delta helps stablize the early
+                stopping strategy against tiny fluctuation, e.g. 0.5->0.49->0.48->0.499->0.500001 is still considered as
+                a good timing for early stopping.
+            early_stop_baseline : float, default=0.0
```

Comment:
Should we have these disabled (None) by default? This seems metric specific. RMSE / r2 / AUC / custom metrics may differ in min/max values.